pages/Documents/ConversationalAI/ConversationOrchestrator/DynamicRouting/getting-started-legacy.md

```diff
@@ -663,7 +663,7 @@ Now test a conversation that doesn’t trigger the complaint routing.
 
 ### Using Conversation Orchestrator outside of Conversational Cloud
 
-This section of the documentation assumes that you are already familiar with linking third-party bots to Conversation Cloud. If you aren't, we strongly suggest that you read the documentation [here](third-party-bots-getting-started.html).
+This section of the documentation assumes that you are already familiar with linking [third-party bots](third-party-bots-getting-started.html) to Conversation Cloud.
```

Comment:
Please change to “Conversational Cloud,” unless we also speak of “Conversation Cloud.” (Not sure, but the former seems to be our preferred wording. We can look into a wider Search and Replace with the team.)